pxtlib/diff.ts

```diff
@@ -105,7 +105,7 @@ namespace pxt.diff {
             let y = x - k
             const snakeLen = V[MAX + k] - x
             for (let i = snakeLen - 1; i >= 0; --i)
-                diff.push("  " + a[x + i])
+                diff.push("  " + b[y + i])
```

Comment:
does this make sense to take file b's content at the matching line instead of file a's? need that or you get the spacing from the previous step, which will be wrong when the spacing changed / etc - very problematic when using it for step diffs, as it will show up like this:

<img width="632" alt="Screen Shot 2020-02-11 at 5 47 26 PM" src="https://user-images.githubusercontent.com/5615930/74299240-e2e1a980-4d00-11ea-8f89-a747fafea414.png">

instead of

<img width="603" alt="Screen Shot 2020-02-11 at 7 02 47 PM" src="https://user-images.githubusercontent.com/5615930/74299314-1e7c7380-4d01-11ea-964a-bb2dece2f1de.png">

that change seems to fix it but i haven't looked too deeply at the diff algorithm / it's pretty dense

Reply:
looks ok. good catch